### PR TITLE
Previous functionality

### LINKED ISSUE #31: User can view previously found restaurants
- [x] User should be able to view a list of previously found restaurants under the previous tab. 
- [ ] STRETCH: User should be able to press on a previous restaurant and view all information about the restaurant.

### PR DESCRIPTION
**Pull request by CHaiz15**
## What functionality does this accomplish?
closes #31 

**Description:**
Added AsyncStorage to store all of the users previously fetched restaurants. 
The user is able to pull down on the page to refresh or refresh the app to see their most recent restaurants.

## What did you struggle on to complete?
Correctly using AsyncStorage to store an array took many hours of googling and learning to complete.

### Test Coverage Percentage: x%
- [x] No Tests have been changed
- [ ] New Tests have been implemented
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

## Please include a gif of how you feel about this branch:
Storage gif

### CODE REVIEW

File: src/screens/asyncStorageHelper.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { AsyncStorage } from 'react-native';
```
Comment (kimmccaskill) [inline src/screens/asyncStorageHelper.js:1]: Some of this looks like something I know... 🧐

File: src/screens/tabs/PrevTab.js [hunk 2]
```diff
@@ -1,170 +1,78 @@
...
-    price: '$',
-    rating: '2.1',
-  },
-  {
-    name: 'McDonalds',
-    price: '$',
-    rating: '0.8',
-  },
-  {
-    name: 'Casa Bonita',
```
Comment (kimmccaskill) [inline src/screens/tabs/PrevTab.js:126]: RIP in peace

File: src/screens/tabs/PrevTab.js [hunk 3]
```diff
@@ -1,170 +1,78 @@
...
+      {isLoading ? <ActivityIndicator size='large' color='blue'/> : 
+        <>
+          <ScrollView showsVerticalScrollIndicator={false}>
+          {previous.length === 0 
+            ? 
+            <View style={styles.noPrevious}>
+                <Text style={{fontSize: 25}}>You don't have any previous 🥺</Text>
+              </View>
+            :
+            previous.reverse().map(restaurant => {
```
Comment (kimmccaskill) [inline src/screens/tabs/PrevTab.js:56]: 🤫

File: src/screens/tabs/PrevTab.js (hunk 2 above)
Comment (CHaiz15) [reply src/screens/tabs/PrevTab.js:126]: May she live on forever 😪

**Pull request merged**